docs/README.md

```diff
@@ -70,7 +70,7 @@ There are 3 main components in the hub page: 1. Stage title, 2. Row of Card butt
 
     <a class="card" href="/articles/request-money/SmartScan">
         <div class="body">
-            <div class="title">SmartScan</div>
+            <h3>SmartScan</h3>
```

Comment:
NAB, but it's best to keep markup decoupled from styles, so I would have kept the `class=title` to apply the styles and only changed the tag. That way if you needed an h3 somewhere else (for document hierarchy purposes), but you wanted it to look different, you don't need to override a bunch of things or pick a less appropriate HTML tag

Reply:
Made this change